src/harness/reference_models/geo/utils_test.py

```diff
@@ -159,6 +159,20 @@ def test_polygons_equal(self):
     poly = sgeo.Point(0, 0.1).buffer(1)
     self.assertTrue(utils.PolygonsAlmostEqual(poly_ref, poly, 15))
 
+  def test_polygon_correct_geoJson_winding(self):
```

Comment:
`geojson` (the `j` should not be capitalized).

Reply:
fixed, thanks.